dsl/src/main/java/tech/pegasys/peeps/SignerConfiguration.java

```diff
@@ -10,32 +10,26 @@
  * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package tech.pegasys.peeps;
+package tech.pegasys.peeps.signer;
 
-import tech.pegasys.peeps.node.Account;
 import tech.pegasys.peeps.signer.model.SignerIdentifier;
 import tech.pegasys.peeps.signer.model.SignerKeyFileResource;
 import tech.pegasys.peeps.signer.model.SignerPasswordFileResource;
 import tech.pegasys.peeps.signer.model.WalletFileResources;
 
 import org.apache.tuweni.eth.Address;
 
-public enum SignerConfiguration {
-  ALPHA(
-      "signer/account/funded/wallet_a.v3",
-      "signer/account/funded/wallet_a.pass",
-      Account.ALPHA.address()),
-  BETA(
-      "signer/account/funded/wallet_b.v3",
-      "signer/account/funded/wallet_b.pass",
-      Account.BETA.address());
+public class SignerConfiguration {
 
   private final Address address;
   private final SignerIdentifier id;
   private final WalletFileResources resources;
 
-  SignerConfiguration(
-      final String keyResource, final String passwordResource, final Address address) {
+  public SignerConfiguration(
+      final String name,
```

Comment:
should this be id instead of name? id is being used elsewhere

Reply:
changed it all over to be "name" and removed SignerIdentifier all together.